examples/column_transformer.py

```diff
@@ -128,41 +89,34 @@ def transform(self, posts):
     ('subjectbody', SubjectBodyExtractor()),
 
     # Use FeatureUnion to combine the features from subject and body
-    ('union', FeatureUnion(
-        transformer_list=[
-
-            # Pipeline for pulling features from the post's subject line
-            ('subject', Pipeline([
-                ('selector', ItemSelector(key='subject')),
-                ('tfidf', TfidfVectorizer(min_df=50)),
-            ])),
+    ('union', ColumnTransformer(
+        {
+            # Pulling features from the post's subject line
+            'subject': (TfidfVectorizer(min_df=50), 'subject'),
 
             # Pipeline for standard bag-of-words model for body
-            ('body_bow', Pipeline([
-                ('selector', ItemSelector(key='body')),
+            'body_bow': (Pipeline([
                 ('tfidf', TfidfVectorizer()),
                 ('best', TruncatedSVD(n_components=50)),
-            ])),
+            ]), 'body'),
```

Comment:
While we're at it, why is the outer structure a dict and not an ordered list of tuples like FeatureUnion?